aspnetcore/client-side/libman/libman-vs.md

```diff
@@ -0,0 +1,201 @@
+* A subset of [Lodash](https://lodash.com/) was approved by a governing body within the organization. The *lodash.js* and *lodash.min.js* files are retrieved from the local file system at *C:\\tmp\\*. The files are copied to the project's *wwwroot/lib/lodash* folder.
+
+[!code-json[](samples/LibManSample/libman.json)]
+
+> [!NOTE]
+> LibMan only supports one version of each library from each provider. The *libman.json* file fails schema validation if it contains two libraries with the same library name for a given provider.
+
+## Restore library files
+
+Restoration of library files in Visual Studio requires a valid *libman.json* file in the project root. The files are placed in the project at the location specified for each library. Library files can be restored in an ASP.NET Core project in two different ways:
+
+1. [Restore files manually](#restore-files-manually)
+1. [Restore files during build](#restore-files-during-build)
+
+### Restore files manually
+
+To trigger a restore operation for all projects in the solution, select the **Restore Client-Side Libraries** option from the solution-level context menu. To trigger a restore operation for a single project, right-click its *libman.json* file in **Solution Explorer** and select **Restore Client-Side Libraries**.
+
+When the restore operation runs:
+
+* The Task Status Center icon on the Visual Studio status bar is animated. Clicking the icon opens a tooltip listing the known background tasks.
+* Messages are sent to the status bar and the **Library Manager** feed of the **Output** window. For example:
+
+  ```console
+  Restore operation started...
+  Restoring libraries for project LibManSample
+  Restoring library jquery@3.3.1... (LibManSample)
+  wwwroot/lib/jquery/jquery.min.js written to destination (LibManSample)
+  wwwroot/lib/jquery/jquery.js written to destination (LibManSample)
+  wwwroot/lib/jquery/jquery.min.map written to destination (LibManSample)
+  Restore operation completed
+  1 libraries restored in 2.32 seconds
+  ```
+
+### Restore files during build
+
+LibMan can restore the defined library files as part of the build process. By default, the restore-on-build behavior is disabled. To enable and test it:
+
+* Right-click *libman.json* in **Solution Explorer** and select **Enable Restore Client-Side Libraries on Build** from the context menu.
+* Click the **Yes** button when prompted to install a NuGet package. The [Microsoft.Web.LibraryManager.Build](https://www.nuget.org/packages/Microsoft.Web.LibraryManager.Build/) NuGet package is added to your project:
+
+  [!code-xml[](samples/LibManSample/LibManSample.csproj?name=snippet_RestoreOnBuildPackage)]
+
+* Build the project to confirm LibMan file restoration occurs. The `Microsoft.Web.LibraryManager.Build` package injects an MSBuild target that runs LibMan during the project's build operation.
```

Comment:
If interested in linking ...

https://www.nuget.org/packages/Microsoft.Web.LibraryManager.Build/

Reply:
I linked to it a few lines up, so I felt that was unnecessary to do again.

Reply:
I missed it.